examples/step-85/step-85.cu

```diff
@@ -58,10 +66,18 @@ namespace Step85
       : coef(coefficient)
     {}
 
+    // Later, we wan't to pass an object of this type to a
```

Comment:
wan't -> want Also did you mean to have the same text copy and paste a few lines below